packages/core/src/components/hotkeys/hotkeyParser.ts

```diff
@@ -109,6 +109,11 @@ export const parseKeyCombo = (combo: string): KeyCombo => {
                 Valid key combos look like "cmd + plus", "shift+p", or "!"`);
         }
 
+        if (piece === "esc") {
+            piece = "escape";
+            piece = CONFIG_ALIASES[piece];
```

Comment:
IMO, an easier way to solve this would be to add an entry to the `CONFIG_ALIASES` object:

```ts
    esc: "escape",
```